File: testData/src/java8/pkg/TestTryFinally.java
package pkg;

public class TestTryFinally {
  public void test0() {
    try {
      System.out.println("Hello");
    } finally {
      long l = 5;
    }
  }

  public void test1() {
    try {
      System.out.println("Hello");
    } finally {
      System.out.println("Finally");
    }

    System.out.println("Bye");
  }


  public void test2(int i) {
    try {
      System.out.println("Hello");
    } finally {
      System.out.println("Finally");
      if(i > 0) {
        System.out.println(i);
        return;
      }
    }

    System.out.println("Bye");
  }
}

File: plugins/kotlin/testData/src/kt/pkg/TestTryFinally.kt
package pkg

class TestTryFinally {
    fun test0() {
        try {
            println("Hello")
        } finally {
            val l: Long = 5
        }
    }

    fun test1() {
        try {
            println("Hello")
        } finally {
            println("Finally")
        }

        println("Bye")
    }


    fun test2(i: Int) {
        try {
            println("Hello")
        } finally {
            println("Finally")
            if (i > 0) {
                println(i)
                return
            }
        }

        println("Bye")
    }
}
